src/screens/General/Splash.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import {
+	View,
+	Image,
+	Text,
+	StyleSheet,
+	Animated,
+	StatusBar
+} from "react-native";
+
+/* Logo */
+import Logo from "../../assets/images/SHPE_UCF_Logo.png";
+
+export class Splash extends Component {
+	state = {
+		LogoAnime: new Animated.Value(0),
+		LogoText: new Animated.Value(0)
+	};
+
+	componentDidMount() {
+		const { LogoAnime, LogoText } = this.state;
+
+		Animated.parallel([
+			Animated.spring(LogoAnime, {
+				toValue: 1,
+				tension: 10,
+				friction: 3,
+				duration: 2000,
+				useNativeDriver: true
+			}).start(),
+
+			Animated.timing(LogoText, {
+				toValue: 1,
+				duration: 2500,
+				useNativeDriver: true
+			})
+		]).start(() => {
+			setTimeout(this.switchScreen, 1500);
+		});
+	}
```

Comment:
I will try to fix the `Animated` custom class to make this simpler as the purpose was to avoid importing the react native animations in multiple places